docker-compose.yaml

```
@@ -49,7 +46,18 @@ services:
     volumes:
       - ./local_config/pgadmin/:/var/lib/pgadmin/
     depends_on:
-      - db
+      db:
+        condition: service_healthy
+
+  alembic:
+    build:
+      context: .
+      target: alembic-image
+    environment:
+      - SQLALCHEMY_DATABASE_URI=${SQLALCHEMY_DATABASE_URI:-postgresql://postgres:postgres@metadata_db_1:5432/metadata}
```

As I understand it won't work in most cases with `metadata_db_1`:
```suggestion
      - SQLALCHEMY_DATABASE_URI=${SQLALCHEMY_DATABASE_URI:-postgresql://postgres:postgres@db:5432/metadata}
```

---

I modified Alembic's config to not use an environment variable for its database connection.